constants.yml

```diff
@@ -4,6 +4,7 @@ identification_type:
     'eea' : "European Economic Area"
     'uk-limited-company' : "UK Limited Company"
     'other-corporate-body-or-firm' : "Other Corporate Body or Firm"
+    'overseas-entity-corporate-managing-officer' : "Corporate managing officer"
```

Comment:
Does this need changing to `registered-overseas-entity-corporate-managing-officer`? See the updated version of point 6 on https://companieshouse.atlassian.net/browse/ROE-174.